sdk/spring/spring-security-azure-oauth2-client/src/main/java/com/azure/spring/security/oauth2/client/implementation/constants/Constants.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.security.oauth2.client.implementation.constants;
+
+/**
+ * Constants used for Azure AD related logic.
+ */
+public interface Constants {
```

Comment:
can we give this a more specific name instead of this general one, or we will have like hundred classes named same

Reply:
Deleted.